sunpy/io/special/asdf/entry_points.py

```diff
@@ -37,12 +37,22 @@ def get_extensions():
     from sunpy.io.special.asdf.converters.generic_map import GenericMapConverter
 
     sunpy_converters = [GenericMapConverter()] + SUNPY_FRAME_CONVERTERS
+    # order here is important as asdf will pick the first matching
+    # extension, start with the newest
+    _manifest_uris = [
+#        "asdf://sunpy.org/sunpy/manifests/sunpy-1.2.0",
+#        "asdf://sunpy.org/sunpy/manifests/sunpy-1.1.0",
```

Comment:
With these two, all of the asdf tests fail. 
If I leave it at 1.0.0, most of them pass but there are still fails.